docs/TELEMETRY_SPECIFICATION.md

```
@@ -0,0 +1,1167 @@
+
+#### 2.2.1 API Entry (SERVER span)
+
+**When**: HTTP request received at REST API endpoint
+
+**Span Name**: `API {HTTP_METHOD} {PATH}`
+
+**Required Attributes**:
+| Attribute | Type | Example |
+|-----------|------|---------|
+| `http.method` | string | "POST" |
+| `http.url` | string | "http://localhost:8080/v1/vm-instances" |
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)